app/services/import_countries_service.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require 'open-uri'
+
+class ImportCountriesService
+  # Ignore these invalid/obsolete countries:
+  INVALID_COUNTRY_CODES = %w[CS XK AQ UM] # Serbia and Montenegro, Kosovo (not there yet), Antartica
+
+  class CountryData < Struct.new(:code, :name, :currency)
```

Comment:
Avoid inheriting from Struct.new:

- Struct.new creates an anonymous class
- Then you inherit from this anonymous class
- also one struct is equal to another if attributes are equal, they are more like data containers not real object you want to work with.

```ruby
Foo = Struct.new(:bar)
foo1 = Foo.new('bar')
foo2 = Foo.new('bar')

{foo1 => :a, foo2 => :b}.length
#=> 1
```

/cc @3scale/system for your knowledge 🔝 

You might expect something else